CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. 
+
+# Pull Requests
+
+Pull requests should be named with the format `crate: short description of change`, and should use
+lower case letters. Always make a pull request from a new branch, that is named similarly, but with
```

Comment:
> ... and should use lower case letters

Does this include proper nouns like Discord, Redis, etc?

Reply:
I think so, yes.

Reply:
I'm not a fan, but I'll let others throw their opinions into the hat on this one.